src/pcap/config/config.example.yml

```diff
@@ -0,0 +1,56 @@
+agent:
```

Comment:
(this is a shortcoming of me rotating out. I was planning to create some test that reads in the config, so it would verify the sample file. This was not done yet

Reply:
The file was something like a basis for discussion about config. I suggest the finalizing config and tests for that also as the next step.